contracts/tokenbridge/ethereum/gateway/L1OrbitERC20Gateway.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.0;
+
+import { L1ERC20Gateway, IERC20 } from "./L1ERC20Gateway.sol";
+import { IERC20Inbox } from "../L1ArbitrumMessenger.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20Bridge } from "../../libraries/IERC20Bridge.sol";
+
+/**
+ * @title Layer 1 Gateway contract for bridging standard ERC20s in ERC20-based rollup
+ * @notice This contract handles token deposits, holds the escrowed tokens on layer 1, and (ultimately) finalizes withdrawals.
+ * @dev Any ERC20 that requires non-standard functionality should use a separate gateway.
+ * Messages to layer 2 use the inbox's createRetryableTicket method.
+ */
+contract L1OrbitERC20Gateway is L1ERC20Gateway {
+    using SafeERC20 for IERC20;
+
+    function outboundTransferCustomRefund(
+        address _l1Token,
+        address _refundTo,
+        address _to,
+        uint256 _amount,
+        uint256 _maxGas,
+        uint256 _gasPriceBid,
+        bytes calldata _data
+    ) public payable override returns (bytes memory res) {
+        require(_l1Token != _getNativeFeeToken(), "NOT_ALLOWED_TO_BRIDGE_FEE_TOKEN");
```

Comment:
I think a comment here about why this isnt allowed would be good

Reply:
done